src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProcessor.cs

```diff
@@ -35,59 +73,130 @@ public ConsoleLoggerProcessor(IConsole console, IConsole errorConsole)
 
         public virtual void EnqueueMessage(LogMessageEntry message)
         {
-            if (!_messageQueue.IsAddingCompleted)
-            {
-                try
-                {
-                    _messageQueue.Add(message);
-                    return;
-                }
-                catch (InvalidOperationException) { }
-            }
-
-            // Adding is completed so just log the message
-            try
+            // cannot enqueue when adding is completed
+            if (!Enqueue(message))
             {
                 WriteMessage(message);
             }
-            catch (Exception) { }
         }
 
         // for testing
         internal void WriteMessage(LogMessageEntry entry)
         {
-            IConsole console = entry.LogAsError ? ErrorConsole : Console;
-            console.Write(entry.Message);
+            try
+            {
+                IConsole console = entry.LogAsError ? ErrorConsole : Console;
+                console.Write(entry.Message);
+            }
+            catch (Exception)
```

Comment:
If we really want to eat everything, this can just be:
```suggestion
            catch
```